src/main.cpp

```diff
@@ -1590,6 +1590,18 @@ static void AlertNotify(const std::string& strMessage, bool fThread)
         runCommand(strCmd);
 }
 
+static void AlertNotifyOnce(const std::string& strMessage, bool fThread)
```

Comment:
If you are hardcoding this to `true`, there is no reason to keep it in the args. Am I missing something?